developer/src/common/web/utils/src/types/keyman-touch-layout/keyman-touch-layout-file-writer.ts

```diff
@@ -1,4 +1,8 @@
-import { TouchLayoutFile, TouchLayoutPlatform, TouchLayoutKey, TouchLayoutSubKey } from "./keyman-touch-layout-file.js";
+import { TouchLayout } from "@keymanapp/common-types";
```

Comment:
OK, glad to see the core definitions for TouchLayout typing remain in common/web/types; Web does use them.